runtimes/php/php-apps.md

```diff
@@ -446,6 +436,16 @@ monolog:
 
 You can change the level to whatever level you desire. For Symfony, the configuration file is `app/config/config_prod.yml`.
 
+Laravel doesn't need Monolog to retrieve logs via Clever console or Clever CLI. Here, ensure that you have the following line in `config/app.php`:
+
+```php
+...
+'log' => env('APP_LOG', 'syslog'),
+...
+```
+
+Then, set `APP_LOG=syslog` as Clever application environment variable.
```

Comment:
I used the official getting started of laravel and IIRC that's the most simple way to configure logs.